client/stylesheets/_responsive.scss

```diff
@@ -509,6 +538,20 @@
     
 }
 
+@media (max-width: 350px) {
+    .profileCardInfo li:first-child {
+        margin-bottom: 4px !important;
+    }
+
+    .profileCardItem {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardItem` should be written in lowercase with hyphens